tests/taskcluster/tasks.yml

```diff
@@ -33,6 +33,8 @@ definitions:
   marionette_js_tests:
     chunks: 30
 
+  marionette_js_tv_tests:
+
```

Comment:
Currently we don't have much tests to run on tv, we could add chunks back when tv tests grow.